src/deploy/functions/backend.ts

```diff
@@ -228,7 +228,7 @@ export function memoryToGen2Cpu(memory: MemoryOptions): number {
     2048: 1,
     4096: 2,
     8192: 2,
-    16384: 3,
+    16384: 2,
```

Comment:
Should have been 4 according to a quick survey where we create a 16GB function in the Cloud Console